tests/integration_tests/modules/test_write_files.py

```diff
@@ -84,6 +86,14 @@ class TestWriteFiles:
         ),
     )
     def test_write_files(self, cmd, expected_out, class_client):
+        if class_client.execute("command -v file").failed:
+            if CURRENT_RELEASE.os not in ("ubuntu", "debian"):
+                pytest.skip(
+                    f"Image {CURRENT_RELEASE.os} doesn't contain 'file' util"
+                )
+
+            class_client.execute("apt-get update")
```

Comment:
Is it worth updating this test to use something other than `file`? That particular command doesn't seem to be integral to the test.

Reply:
Good point, we don't need to use `file` for this verification/validation. Simplifying this to just check md5sums of the output file.